packages/react-components/src/components/VideoTile.tsx

```diff
@@ -156,32 +156,34 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           </Stack>
         )}
 
-        <Stack
-          horizontal
-          className={mergeStyles(
-            isVideoRendered ? videoHint : disabledVideoHint,
-            // when video is being rendered, the displayName has a grey-ish background, so no use of theme
-            { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary },
-            styles?.displayNameContainer
-          )}
-        >
-          <Stack>
-            {displayName && (
-              <Text
-                className={mergeStyles(displayNameStyle, {
-                  color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary
-                })}
-              >
-                {displayName}
-              </Text>
+        {(displayName || showMuteIndicator) && (
```

Comment:
The logic here is dense, and probably slightly wrong?

Should this be:

    (displayName || (showMuteIndicator && isMuted !== undefined))

It's probably more readable, and not all that wrong, to unconditionally create this `Stack`. If all the contents end up being skipped, it will be an empty `Stack`. It'll be hard to keep the conditionals correctly synchronized as it stands.